### DISCUSSION TITLE
Solved: {{file_name} automatically puts ".md" in the end

### DISCUSSION THREAD

**Discussion by m00nlygreat** [Taitava/obsidian-shellcommands#400] [Debugging and testing]
even if there is no ".md" in the real file name. seems like hard-coded

**Comment by Taitava** [comment]
Hi. Is it a file without an extension or does it have some other extension, e.g. `.pdf`?

**Comment by m00nlygreat** [comment]
Sorry, it was my mistake. the file had extension. I knew it just after writing this issue but I didn't know how I find this thread 🙄

**Reply by Taitava** [reply]
Ok, no problem. 🙂👍
